src/controllers/user.ts

```diff
@@ -12,9 +12,17 @@ router.get("/:id", (req, res) => {
 })
 
 router.post("/", (req, res) => {
-    const user = new User({ email: 'controller-test', firstName: 'controller-test' })
-    console.log(user, "controller was hit")
-    res.sendStatus(204)
+    const user = new User({
+        email: req.body.email,
+        firstName: req.body.firstName,
+        lastName: req.body.lastName,
+        address: req.body.address,
+        dateOfBirth: req.body.dateOfBirth
+    })
```

Comment:
We could write this as 
```js
const {
  email,
  firstName,
  lastName,
  address,
  dateOfBirth
} = req.body;

const user = new User({
  email,
  firstName,
  lastName,
  address,
  dateOfBirth
});
```
This will be cleaner and more flexible in the long run.